css/style.css

```diff
@@ -1039,22 +1041,1372 @@ li { list-style-type: none; }
+    text-align: left;
+    margin: -22px 0 0 64px;
+    padding: 0;
+  }
+
+  .button {
+    display: flex;
+    width: 220.84px;
+    height: 55.21px;
+    font-family: "Montserrat", sans-serif;
+    text-align: center;
+    font-weight: bold;
+    text-transform: uppercase;
+  }
+  
+  .test-drive {
+    display: flex;
+    flex-flow: row wrap;
+    width: 418.21px;
+    height: 55.21px;
+    justify-content: flex-start;
+    align-items: center;
+    margin: 0 34.53px 6px 34.53px;
+  }
+
+  .test-drive__rudder {
+    display: flex;
+    width: 30.37px;
+    height: 30.37px;
+    margin: 0 9px 0 0;
+    padding: 0;
+    background: url(https://i.ibb.co/Stm3197/2-7-red-wheel.png) no-repeat;
+    background-size: 100%;
+    background-position: center;
+  }
+
+  .test-drive__link {
+    display: flex;
+    justify-content: center;
+    width: 122.84px;
+    height: 24.84px;
+    font-size: 16px;
+    line-height: 20px;
+    font-style: normal;
+    font-weight: bold;
+    text-decoration: none;
+    color: #e61234;
+    margin: 0 34.51px 0 0;
+    text-transform: uppercase;
+  }
+
+  .test-drive__link:hover {
+    display: flex;
+    font-size: 16px;
+    line-height: 20px;
+    font-weight: bold;
+    text-align: center;
+    text-decoration: none;
+    color: #000;
+    cursor: pointer;
+    padding: 0;
+    border: 0;
+  }  
+
+  .test-drive__button {
+    display: flex;
+    width: 221px;
+    height: 60px;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 16px;
+    line-height: 20px;
+    text-align: center;
+    font-weight: bold;
+    border: 2px solid #fff;
+    color: #fff;
+    background-color: #f00;
+  }
+  
+  .test-drive__button:hover {
+    font-size: 14px;
+    color: #fff;
+    border: 3px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+  }
+  
+  .test-drive__button:focus {
+    display: flex;
+    color: #fff;
+    border: 1px solid #fff;
+    background-color: #f00;
+    cursor: pointer;
+    font-weight: bold;
+    padding: 0;
+    outline: none;
+    box-shadow: 0 0 0 3px;
+  }
+  
+  .test-drive__button:active {
+    background-color: #fff;
+    border-color: #000;
+    color: #000;
+    font-size: 12px;
+  }
+  
+  .section__btn-pointer {
+    display: flex;
+    width: 700px;
+    height: 20px;
+    padding: 0;
+    margin: 0 auto 61px auto;
+    box-shadow: none;
+    background: url(https://i.ibb.co/GFX8Xs7/Vector-1.png) no-repeat;
+    background-size: 5%;
+    background-position: center center;
+    outline: none;
+  }
+
+  /* -------2-------- */
+
+  .sections__dashboard {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 768px;
+    height: 988.03px;
+    align-content: flex-end;
+    margin: 0;
+    background: url(https://i.ibb.co/JkTYWY9/2013-tesla-model-s-cockpit-i-pad.png) no-repeat;
+    background-size: 100.5% 51%;
+    background-position: top left;
+   }
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)